### DISCUSSION TITLE
shift+arrow .  does not select the way it used to - need counsel

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#1163] [Help] [answered]
I kind of took a break from Freeplane for a couple of years.  Back then, pressing shift+arrow on a node selected then node I was on plus displayed nodes displayed further down the branch.  Now I cannot figure out what it is designed to do or how to accomplish what it used to do without a bunch of ctrl+click.

The attached video shows this.

I used to be very pleased with the way FP worked in this regard.  Now I'm just confused and frustrated.

Any help would be appreciated.  If no fix is available, I'd be interested in the rationale for such a change.

[shift arrow selection.zip](https://github.com/freeplane/freeplane/files/11526002/shift.arrow.selection.zip)

**Comment by wordmuse** [comment, marked as answer]
I think you are talking about the command _Select Visible Branch_ on the _Navigation_ menu. If you mouse over to that and press Ctrl while clicking the mouse, you should get a dialog that lets you assign your preferred keyboard combination. I use ctrl+shift+A. You could assign it what you prefer. Hope this helps.

~ Bal

**Reply by kenfhill84083** [reply]
> I think you are talking about the command _Select Visible Branch_ on the _Navigation_ menu. If you mouse over to that and press Ctrl while clicking the mouse, you should get a dialog that lets you assign your preferred keyboard combination. I use ctrl+shift+A. You could assign it what you prefer. Hope this helps.
> 
> ~ Bal

Thanks.  That is what used to happen.  I guess I should go browsing thru the menus.

**Reply by ls-1N** [reply]
This works, but the original Shift + arrow keys was directional. I.e. if the child nodes were on the right I pressed Shift+Right_arrow and if the nodes were left, I would press Shift+Left_arrow. So it feels different and the habit will need to change slightly. But the new system might be an improvement, since those who freely reposition their nodes, might end up in situations where it is ambiguous whether a right or left arrow is needed.

**Reply by euu2021** [reply]
> This works, but the original Shift + arrow keys was directional. I.e. if the child nodes were on the right I pressed Shift+Right_arrow and if the nodes were left, I would press Shift+Left_arrow. So it feels different and the habit will need to change slightly.

macmarrum has many scripts for selection. I think it's possible to achieve any behavior you may want:
![image](https://github.com/freeplane/freeplane/assets/77707706/85889d9d-fdce-4259-a9ed-7e3779469aad)

See his scripts at: https://github.com/macmarrum/freeplane-scripting/tree/main/src/scripts
